src/Symfony/Component/Messenger/Stamp/RedeliveryStamp.php

```diff
@@ -21,23 +20,11 @@ final class RedeliveryStamp implements StampInterface
 {
     private $retryCount;
     private $redeliveredAt;
-    private $exceptionMessage;
-    private $flattenException;
 
-    public function __construct(int $retryCount, string $exceptionMessage = null, FlattenException $flattenException = null, \DateTimeInterface $redeliveredAt = null)
+    public function __construct(int $retryCount, \DateTimeInterface $redeliveredAt = null)
```

Comment:
It looks like the BC layer is broken in 5.2: passing $redeliveredAt as the second argument should be allow already.
Can you please submit a PR doing so?

Reply:
Sure. #42059